src/template/template-parts/tab.php

```diff
@@ -47,18 +47,83 @@
         </p>
       </div>
     </div>
+     <?php if (is_page([93])): ?>
```

Comment:
This cannot be.
The ids of pages in the server and in your local machine are different and this won't work.
Please use the page name instead